src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromPartner.tsx

```diff
@@ -134,11 +134,10 @@ export const ArtworkDetailsAboutTheWorkFromPartnerFragmentContainer = createFrag
           city
         }
         profile {
-          __id
+          ...FollowProfileButton_profile
           id
-          is_followed
           icon {
-            url
+            url(version: "square140")
```

Comment:
Also noticed that on the site this image looked off compared to what was on partner details before. Realizing we need a properly square version rather then resizing the large one.